Jenkinsfile

```diff
@@ -0,0 +1,117 @@
+import groovy.json.JsonOutput
+
+/** Tox environment */
+def environment = 'e2e'
+
+/** Run Tox
+ *
+ * @param environment test environment to run
+*/
+def runTox(environment) {
+  def options = env.PYTEST_ADDOPTS ?: ''
```

Comment:
There's no guarantee this environment variable is set, so this line sets a default of an empty string.